open-api/rest-catalog-open-api.yaml

```diff
@@ -2068,6 +2162,145 @@ components:
+        partition:
+          type: object
+          additionalProperties:
+            type: string
+        size-bytes:
+          type: number
+        start:
+          type: number
+        length:
+          type: number
+        estimated-rows-count:
+          type: number
+        delete-files:
+          type: array
+          items:
+            $ref: '#/components/schemas/ContentFile'
+        schema:
+          $ref: '#/components/schemas/Schema'
+        spec:
+          $ref: '#/components/schemas/PartitionSpec'
+        residual-filter:
+          $ref: '#/components/schemas/Expression'
+
+    TypeValue:
+      oneOf:
+        - $ref: '#/components/schemas/PrimitiveTypeValue'
+        - $ref: '#/components/schemas/MapTypeValue'
+        - $ref: '#/components/schemas/StructTypeValue'
+
+    MapTypeValue:
+      type: object
+      properties:
+        keys:
+          type: array
+          items:
+            $ref: '#/components/schemas/TypeValue'
+        values:
+          type: array
+          items:
+            $ref: '#/components/schemas/TypeValue'
+
+    StructTypeValue:
+      type: object
+      additionalProperties:
+        oneOf:
+          - $ref: '#/components/schemas/TypeValue'
+
+    PrimitiveTypeValue:
+      oneOf:
+        - type: boolean
+        - type: integer
+        - type: integer
+          format: int64
+        - type: number
+        - type: string
+        - type: string
+          format: byte
+        - type: array
+          items:
+            $ref: '#/components/schemas/TypeValue'
+
+    ContentFile:
+      type: object
+      required:
+        - spec-id
+        - content
+        - file-path
+        - file-format
+        - file-size-in-bytes
+        - record-count
+      properties:
+        spec-id:
+          type: integer
+        content:
+          $ref: '#/components/schemas/FileContent'
+        file-path:
+          type: string
+        file-format:
+          type: string
+        partition:
+          $ref: '#/components/schemas/StructTypeValue'
+        file-size-in-bytes:
+          type: integer
+          format: int64
+        record-count:
+          type: integer
+          format: int64
+        column-sizes:
+          $ref: '#/components/schemas/MapTypeValue'
+        value-counts:
+          $ref: '#/components/schemas/MapTypeValue'
+        null-value-counts:
+          $ref: '#/components/schemas/MapTypeValue'
+        nan-value-counts:
+          $ref: '#/components/schemas/MapTypeValue'
+        lower-bounds:
+          $ref: '#/components/schemas/MapTypeValue'
+        upper-bounds:
+          $ref: '#/components/schemas/MapTypeValue'
+        key-metadata:
+          type: string
+          format: byte
+        split-offsets:
+          type: array
+          items:
+            type: integer
+            format: int64
+        equality-ids:
+          type: array
+          items:
+            type: integer
+        sort-order-id:
```

Comment:
You can remove this.

Reply:
I can remove this but I see that the sortOrderId is in the ContentFileParser

```
if (contentFile.sortOrderId() != null) {
      generator.writeNumberField(SORT_ORDER_ID, contentFile.sortOrderId());
 }

```

Reply:
Looks like we kept this in Drews ContentFile pr: https://github.com/apache/iceberg/commit/bb53c3d4e0e27ac6706803c2371793ad2476ae04 
so I believe we can keep this?

Reply:
Yeah, I'm okay keeping it.